src/test/java/uk/gov/ons/census/fwmt/jobservice/service/JobCacheManagerTest.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.ons.census.fwmt.jobservice.service;
```

Comment:
delete this tezt, Im not sure it does anything that useful

Reply:
This has been removed.